package.json

```diff
@@ -17,28 +17,28 @@
   },
   "private": true,
   "devDependencies": {
-    "@types/chai": "^4.3.5",
-    "@types/mocha": "^10.0.1",
-    "@types/node": "^18.15.3",
-    "@types/vscode": "^1.77.0",
-    "@typescript-eslint/eslint-plugin": "^5.59.2",
-    "@typescript-eslint/parser": "^5.59.2",
-    "@vscode/debugadapter": "^1.61.0",
-    "@vscode/debugprotocol": "1.61.0",
-    "@vscode/extension-telemetry": "0.8.4",
-    "@vscode/test-web": "^0.0.45",
-    "chai": "^4.3.8",
-    "esbuild": "^0.17.12",
-    "eslint": "^8.39.0",
+    "@types/chai": "^4.3.8",
+    "@types/mocha": "^10.0.2",
+    "@types/node": "^20.8.6",
```

Comment:
I probably shouldn't have bumped the node.js typings. Will downgrade this to 18 to match the `engines`  requirement.